R/findNLDI.R

```diff
@@ -226,52 +240,46 @@ valid_ask = function(all, type){
 #' ## GENERAL ORIGIN: WaDE
 #'  findNLDI(origin = list("wade" = 'CA_45206'))
 #'
-#' # Navigation
+#' # Navigation (flowlines will be returned by default)
```

Comment:
"Flowlines will be returned if `find` is not specified" ??

Reply:
If this is a question, yes, findNLDI(comid = 101, nav = "UT") will currently give the upstream flowlines

Reply:
No -- I was suggesting different documentation.

Reply:
👍